meta-ibm/recipes-phosphor/vpd/ibm-vpd-parser/50001000.json

```diff
@@ -1076,6 +1076,7 @@
     "/sys/bus/i2c/drivers/at24/4-0050/eeprom": [
       {
         "inventoryPath": "/system/chassis/motherboard/pcieslot0/pcie_card0",
+	"pcaChipAddress" : "4-0060",
```

Comment:
nit: the indent doesn't match the rest of the lines

Reply:
These files got added from other commit. Not needed with this commit so I removed